lib/plan-finder.js

```diff
@@ -15,9 +15,14 @@
 //const debug = require('debug')('pipeline:planFinder');
 const debug = console.log;
 const TransformersGraph = require('./transformer-graph');
-const {manifestIntersection, checkInputMatches, bestOfIntersection} = require('./plan-finder-utils');
-const {SourceCorruptError, RenditionFormatUnsupportedError} = require('@adobe/asset-compute-commons');
+const { manifestIntersection, checkInputMatches, bestOfIntersection } = require('./plan-finder-utils');
+const { SourceCorruptError, RenditionFormatUnsupportedError } = require('@adobe/asset-compute-commons');
 const clone = require('clone');
+const fs = require('fs-extra');
+const validUrl = require('valid-url');
+const child_process = require('child_process');
+
+const EXIFTOOL_TIMEOUT_MS = 20 * 1000; // max timeout of 20 seconds
```

Comment:
is this timeout ok for larger files?

Reply:
(Even for large files, metadata for images usually is at the beginning of the file... We must test/verify, but ut should be enough... when we have "good" networks if doing the curl+exiftool trick).